account/key/store.go

```diff
@@ -28,27 +27,39 @@ type Store struct {
 	timeout      time.Duration
 	unlocked     map[string]*keyPair
 	unlockedLock *sync.Mutex
-	storage      db.DB
+	storage      Storage
 }
 
 // NewStore make new instance of keystore
 func NewStore(storePath string, unlockTimeout uint) *Store {
-	const dbName = "account"
-	dbPath := path.Join(storePath, dbName)
-	return &Store{
+	store := &Store{
 		timeout:      time.Duration(unlockTimeout) * time.Second,
 		unlocked:     map[string]*keyPair{},
 		unlockedLock: &sync.Mutex{},
-		storage:      db.NewDB(db.LevelImpl, dbPath),
 	}
+
+	// FIXME: more elegant coding
+	if storage, err := LoadBadgerStorage(storePath); nil == err {
+		store.storage = storage
+	} else {
+		storage, err := NewAergoStorage(storePath)
```

Comment:
This stores the keystore files directly in the `$datadir`, right? Maybe we should add a sub directory like `keystore`?

Reply:
Applied in 596508c5f479e6f05f37c2da8f334796826699b8

Reply:
Applied in 9e4226a9998191014d53cae1bceba1ab725f10d3